Sources/JWTKit/Claims/TenantIDClaim.swift

```diff
@@ -7,12 +7,20 @@
 /// logging or auditing the tenant context of the authenticated user. The value of "tid" is a
 /// case-sensitive string representing a GUID. The presence of this claim and its proper validation
 /// are critical for the security of multi-tenant applications.
-public struct TenantIDClaim: JWTClaim, Equatable {
+public struct TenantIDClaim: JWTClaim, Equatable, ExpressibleByStringLiteral, ExpressibleByNilLiteral {
     /// See ``JWTClaim``.
     public var value: String?
 
     /// See ``JWTClaim``.
```

Comment:
When documenting properties or methods which implement protocol requirements, do _not_ use DocC comments; it suppresses autogeneration of the appropriate reference to the protocol.
```suggestion
    // See `JWTClaim.init(value:)`.
```